src/components/Participants/Participants.js

```diff
@@ -0,0 +1,208 @@
+import React, { useState, useEffect } from "react"; 
+import style from './Participants.module.css';
+
+import Input from '../Input/Input';
+import Button from '../Button/Button';
+
+function Participants(index) {
```

Comment:
Should instance of Participants component be added to App (or you wanted to add some more code before that)?